frontend/components/footer.tsx

```diff
@@ -73,6 +73,7 @@ export default function Footer({ baseUrl }: { baseUrl: string }): JSX.Element {
       </p>
 
       <p>
+        <SpacedA href="/community">Community</SpacedA>
```

Comment:
I've linked this in the footer. Maybe it should also be somewhere more prominent, but I'm not sure where..

Reply:
I think this is probably the best place for this for now :+1: